### DISCUSSION TITLE
identify regulons - custom gene list

### DISCUSSION THREAD

**Discussion by paolo-kunderfranco** [aertslab/SCENIC#244] [R]
Dear All,

I would like to find only active regulons whose targets intersects with a custom gene list.

Is there a way to filter TF datasets prior to run runCorrelation and runGenie3 functions?

Best
Paolo

**Comment by changfeifan666** [comment]
Hi, do you find some ways to do this? In my humble opinion, although I have not gotten one exact anewer of this question, I think the papers from Professor Guanghui Liu may gave us some suggestion. There are some references (https://doi.org/10.1016/j.immuni.2026.02.007, https://doi.org/10.1038/s43587-025-01020-y). They did the SCENIC in R and using only the DEGs to analysis. And I also want to find the key TF regulating a custom gene list. So I wonder whether it's proper if I use the genes including   a custom gene list and a subset of TFs (like those differentially expressed).

Warm regards,
Feifan
<img width="945" height="480" alt="image" src="https://github.com/user-attachments/assets/ae80740f-a5ca-4014-bc20-60c73831dcc3" />
